### PR TITLE
Update login and signup satellite selection

### PR DESCRIPTION
**Pull request by boshevski**

### CODE REVIEW

**Review by mbouzi — APPROVED** [review]
👍

File: layouts/partials/satellites-select/select.html [hunk 1]
```diff
@@ -4,30 +4,9 @@
...
-            <h1 class="functional-area__header__title-area__title">Select a Satellite</h1>
-            <p class="functional-area__header__title-area__info">(Metadata region)</p>
-        </div>
-        <svg class="functional-area__header__info-icon" onclick="openInfoPopup()" width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
-            <circle class="info-icon-circle" cx="9" cy="9" r="9" fill="#C4C4C4"/>
-            <path d="M8 9C8 8.44772 8.44772 8 9 8C9.55228 8 10 8.44772 10 9V13C10 13.5523 9.55228 14 9 14C8.44772 14 8 13.5523 8 13V9Z" fill="white"/>
-            <rect x="8" y="4" width="2" height="2" rx="1" fill="white"/>
-        </svg>
-        <div class="functional-area__header__info-popup" id="info-popup">
-            <img src="/img/satellite.png" alt="satellite image">
```
Comment (VitaliiShpital) [inline layouts/partials/satellites-select/select.html:16]: should we also remove this image?

File: layouts/partials/satellites-select/select.html [hunk 2]
```diff
@@ -4,30 +4,9 @@
 <div class="functional-area">
     <div class="functional-area__header">
         <div class="functional-area__header__title-area">
-            <h1 class="functional-area__header__title-area__title">Select a Satellite</h1>
-            <p class="functional-area__header__title-area__info">(Metadata region)</p>
-        </div>
-        <svg class="functional-area__header__info-icon" onclick="openInfoPopup()" width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (VitaliiShpital) [inline layouts/partials/satellites-select/select.html:10]: openInfoPopup() should be removed from scripts then

File: layouts/partials/satellites-select/select.html [hunk 3]
```diff
@@ -4,30 +4,9 @@
...
-        </svg>
-        <div class="functional-area__header__info-popup" id="info-popup">
-            <img src="/img/satellite.png" alt="satellite image">
-            <p class="functional-area__header__info-popup__text">
-                Satellites are metadata regions that manage projects, buckets and objects on Tardigrade.
-            </p>
-            <a class="functional-area__header__info-popup__link" href="https://documentation.tardigrade.io" target="_blank">
-                Learn More
-            </a>
-            <svg class="functional-area__header__info-popup__close-icon" onclick="closeInfoPopup()" width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (VitaliiShpital) [inline layouts/partials/satellites-select/select.html:23]: closeInfoPopup() should be removed from scripts then

**Pull request merged**